2D Simulation/spec/movesTrackerSpec.js

```diff
@@ -0,0 +1,395 @@
+describe("movesTracker", () => {
+    let imported = require('../movesTracker.js');
```

Comment:
This isn't very important either way, but is there any reason these all aren't `const`s instead of `let`s?

Reply:
No... I honestly forgot that was a part of JavaScript... I'll change them as they come up.